react/features/whiteboard/actions.any.ts

```diff
@@ -0,0 +1,28 @@
+import { IReduxState, IStore } from '../app/types';
+
+import { setWhiteboardOpen } from './actions';
+import { isWhiteboardAllowed, isWhiteboardOpen, isWhiteboardVisible } from './functions';
+
+
+/**
+ * API to toggle the whiteboard.
+ *
+ * @param {Object} state - The redux state.
+ * @returns {Function}
+ */
+export function toggleWhiteboard(state: IReduxState) {
+    return async (dispatch: IStore['dispatch']) => {
+        const isAllowed = isWhiteboardAllowed(state);
+        const isOpen = isWhiteboardOpen(state);
+
+        if (isAllowed) {
```

Comment:
For the else case, could you please add a warn log to notify the client that the operation wasn't allowed?

Reply:
"FORBIDDEN" status is added